<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vrSimpleExampleGUI</class>
 <widget class="QWidget" name="vrSimpleExampleGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vrSimpleExample</string>
  </property>
  <property name="group" stdset="0">
   <string>Scripts</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="_new_scene">
     <property name="text">
      <string>New Scene</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="_load_scene">
     <property name="text">
      <string>Load Scene</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="_create_snapshot">
     <property name="text">
      <string>Create Snapshot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
